gio/Gir.toml

```diff
@@ -715,6 +728,11 @@ manual_traits = ["InputStreamExtManual"]
     manual = true
     doc_trait_name = "InputStreamExtManual"
 
+[[object]]
+name = "Gio.IOModule"
+status = "generate"
+cfg_condition = "unix"
```

Comment:
Why wouldn't that exist on Windows? :) That's how you get a TLS module on Windows, for example.

Reply:
But without `GTypeModule` / `GModule` bindings this is not very useful

Reply:
because the windows CI failed at linking stage